src/UI/walletConnect/WalletConnectLoginContainer/index.tsx

```diff
@@ -43,12 +48,31 @@ export const WalletConnectLoginContainer = ({
     logoutRoute,
     callbackRoute,
     token,
-    shouldLoginUser: true,
+    onLoginRedirect
+  });
+  const [
+    initLoginWithWalletConnectV2,
+    connectExisting,
+    { error: walletConnectErrorV2 },
+    {
+      uriDeepLink: walletConnectDeepLinkV2,
+      walletConnectUri: walletConnectUriV2,
+      wcPairings
+    }
+  ] = useWalletConnectV2Login({
+    logoutRoute,
+    callbackRoute,
+    token,
     onLoginRedirect
   });
   const [qrCodeSvg, setQrCodeSvg] = useState<string>('');
   const isMobileDevice =
     platform?.os?.family === 'iOS' || platform?.os?.family === 'Android';
+  const activePairings =
+    isWalletConnectV2 && wcPairings && wcPairings?.length > 0
```

Comment:
you can skip the middle check, wcPairing?.length > 0 will always be false if wcPairing is null or undefined